corehq/apps/accounting/tests/test_customer_invoicing.py

```diff
@@ -268,75 +260,43 @@ def test_product_line_items_in_yearly_invoice(self):
 
         self.assertEqual(CustomerInvoice.objects.count(), 1)
         invoice = CustomerInvoice.objects.first()
-        self.assertEqual(invoice.balance, Decimal('18000.0000'))
-        self.assertEqual(invoice.account, self.account)
-
-        # There should be two product line items, with 3 months billed for each
-        num_product_line_items = invoice.lineitem_set.get_products().count()
-        self.assertEqual(num_product_line_items, 2)
-        for product_line_item in invoice.lineitem_set.get_products().all():
-            self.assertEqual(product_line_item.quantity, 12)
-
-    def test_subscriptions_marked_do_not_invoice_not_included(self):
```

Comment:
Why was this test removed?

Reply:
Because even if a subscription is marked as "do not invoice", we still should include them. This is how Customer billing account set up. If we don't want to include a domain, we will take it out from that account.